File: app/src/main/java/knf/kuma/changelog/objects/Changelog.java
package knf.kuma.changelog.objects;

import org.jsoup.nodes.Document;
import org.jsoup.nodes.Element;

import java.util.ArrayList;
import java.util.List;

public class Changelog {
    public List<Release> releases;

    public Changelog(Document document) {
        List<Release> list = new ArrayList<>();
        for (Element element : document.select("release")) {
            list.add(new Release(element));
        }
        this.releases = list;
    }
}

File: app/src/main/java/knf/kuma/changelog/objects/Changelog.kt
package knf.kuma.changelog.objects

import org.jsoup.nodes.Document
import java.util.*

class Changelog(document: Document) {
    var releases: MutableList<Release>

    init {
        val list = ArrayList<Release>()
        for (element in document.select("release")) {
            list.add(Release(element))
        }
        this.releases = list
    }
}
